### ISSUE TITLE
Alternative RIDE icon

### ISSUE THREAD

**Issue by FionaDyalog** [Dyalog/ride#763]
When installing both Dyalog and RIDE, the icons are identical. I appreciate that this has been discussed before and it was decided that this was a desirable outcome (more benefits than drawbacks), but I don't believe I'm alone in wishing that I had a choice. On Windows (no idea about other platforms), right-clicking the icon on the desktop and going to the properties presents a "change icon" option. Currenetly there are no alternatives offered when this is done; WIBNI there was an alternative icon included here that could be selected (even if it's the same icon in a different colour that would help!). The default icon could remain as it currently is but those who want to could then differentiate between Dyalog and the RIDE.

**Comment by abrudz** [comment]
I think a RIDE-purple coloured cube with the white Dyalog D might work well:
![image](https://user-images.githubusercontent.com/10961427/132663101-218eee5a-30b4-46fd-bc5e-dff831334c68.png)

**Comment by aplteam** [comment]
I can see plenty of cons an no pro at all. Just saying.

Any way to tell them apart would be an improvement.

**Comment by JasonRivers** [comment]
Perhaps using an R instead of a D would be good, as it's "Ride" :-)

**Comment by dyaandys** [comment]
+1 for Jason's R - then we can stick with the Dyalog corporate orange.  

Having said that, what is the icon trying to indicate?  There are three usage cases that I am aware of:
1. Dyalog IDE on Windows
2. Standalone RIDE where you start RIDE and choose the action
3.  RIDE used as the frontend to Dyalog - where starting RIDE takes you straight into Dyalog with no option. Is this the same as (1) or the same as (2) or is it something different ?

**Comment by mbaas2** [comment]
When thinking about these distinct use cases, it might also make sense to provide an icon for zero footprint RIDE that one could use for shortcuts to URLs that open these.

**Comment by aplteam** [comment]
Fully supported.

**Comment by mkromberg** [comment]
I agree with the conclusion, but I don't think "I think much of this confusion comes from the fact that the Windows version of Dyalog is not separable from its IDE" is correct. The cause of the current architecture is that our original Mac user John "Plea for Simplicity" Scholes didn't want a simple user to have to know about both RIDE and the APL system.

Under Windows, RIDE is always a separate application.

**Comment by mkromberg** [comment]
Oops, hit the wrong button :)

**Comment by xpqz** [comment]
I briefly had hopes for the world's fastest fix!
